src/components/views/dialogs/InviteDialog.tsx

```diff
@@ -760,7 +760,7 @@ export default class InviteDialog extends React.PureComponent<Props, IInviteDial
                 const lookup = await MatrixClientPeg.safeGet().lookupThreePid("email", term, token);
                 if (term !== this.state.filterText) return; // abandon hope
 
-                if (!lookup || !lookup.mxid) {
+                if (!lookup || !("mxid" in lookup)) {
```

Comment:
I'm sure this is valid, but i'm not sure i understand what rule it's conforming to?

Reply:
Its because the return type is `{} | { mxid: string; ... }` so to make TS happy we can't just do `!lookup.mxid` as `{}.mxid` is invalid. So we have to use `in`